AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -3488,6 +3488,20 @@ - (UIView *)preferredFocusedView
 
 #pragma mark - Deprecated
 
+- (NSString *)name
+{
+  ASDN::MutexLocker l(__instanceLock__);
+  return _debugName;
+}
+
+- (void)setName:(NSString *)name
+{
+  ASDN::MutexLocker l(__instanceLock__);
```

Comment:
same, rewrite this to be self.debugName = name —locking and equality check handled in the setter.